docs/prow/prow-monitoring.md

```diff
@@ -0,0 +1,73 @@
+# Prow Cluster Monitoring Setup
+
+This document describes how to install and manage Prow cluster monitoring that is available at https://monitoring.build.kyma-project.io. This document also describes how to create and manage Grafana dashboards.
+
+## Prerequisites
+
+Install the following tools
+
+- Helm v2.11.0
+- kubectl
+
+## Provision a monitoring chart
+
+1. Make sure that kubectl points to the correct cluster.
+   
+   ```bash
+   gcloud container clusters get-credentials {clusterName} --zone={zoneName} --project={projectName}
+   ```
+
+2. Go to [prow/cluster](../../prow/cluster) directory
+
+4. Download dependencies
+   
+   ```bash
+   helm dependency build resources/monitoring
+   ```
+
+3. Install Monitoring chart
+
+   ```bash
+   helm install --name {releaseName} --namespace {namespaceName} resources/monitoring
+   ```
+
+4. Go to Grafana dashboard
+   
+   Grafana dashboard is available at this address https://monitoring.build.kyma-project.io. It may take some time till it will be achievable.
+
+## Authenticate to Grafana
+
+By default Grafana dashboards are visible for anonymous users with read-only access. Only authenticated users are able to create/edit dashboards. To login to Grafana please follow this steps
+
+1. Get `admin` password from cluster
+
+   ```
+   kubectl -n {namespaceName} get secret {releaseName}-grafana -o jsonpath="{.data.admin-password}" | base64 -D
+   ```
+
+2. Go to https://monitoring.build.kyma-project.io/login
+
+3. Provide credentials
+
+   ```
+   Login: admin
+   Password: value from 1 step
+   ```
+
+## Create/edit Grafana dashboards
+
+To create/edit Grafana dashboards you must be logged in. Instruction how to works with Grafana dashboards is available [here](http://docs.grafana.org/guides/getting_started/). The main difference is how to store dashboards in our infrastructure.
+
+Please follow these steps:
+
+1. Export dashboard to JSON format.
+
+2. Store JSON in `prow/cluster/resources/monitoring/dashboards/`.
+
+3. Update Grafana configuration on cluster.
+   
+   ```
+   helm upgrade {releaseName} resources/monitoring --recreate-pods
+   ```
+
+   > **NOTE:** `--recreate-pods` is required as secret with Grafana password is regenerated during upgrade operation and it needs to be populated to grafana.
```

Comment:
```suggestion
   > **NOTE:** `--recreate-pods` is required because the Secret with the Grafana password is regenerated during the upgrade and it needs to be populated to Grafana.
```